TheWire fixup up display of messages

drbob opened the pull request:
Process messages to group replies correctly and sort by age.
Expand message display to include replies.
Add coloring to distinguish between replies and actual msgs.
Enable subscribe / unsubscribe buttons.
Add Selection of which set of groups are visible.
Add missing setNetworkExchangeService calls for photo + wire services.

drbob committed:
TheWire fixup up display of messages

Process messages to group replies correctly and sort by age.
Expand message display to include replies.
Add coloring to distinguish between replies and actual msgs.
Enable subscribe / unsubscribe buttons.
Add Selection of which set of groups are visible.
Add missing setNetworkExchangeService calls for photo + wire services.

defnax commented:
hi i get compile error on master not sure there is something missed to fix?
i has updated my branch with master then the photos compile broken

![image](https://user-images.githubusercontent.com/9952056/77252025-0c6fe800-6c52-11ea-8738-5e39f3d8aede.png)

drbob committed:
Merge branch 'master' into thewire_fix_message_display

drbob committed:
Update photo/wire/wiki interfaces to support Blocking Group API.

Add blocking API Group Interfaces for photo,wire and wiki services.
Add service blocking API calls for createGroup/updateGroup/getGroupData.
Update GUI classes to match new GxsGroupDialog Interfaces.

drbob commented:
This has now been updated to compile and run with master
